modules/issue_tracker/jsx/attachments/attachmentsList.js

```diff
@@ -0,0 +1,240 @@
+/**
+ *  The following file handles displaying attachments
+ *  for the issue being viewed in issue_tracker.
+ *
+ *  @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ *  @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ *  @link     https://github.com/aces/Loris
+ */
+
+import React, {Component, Fragment} from 'react';
+import PropTypes from 'prop-types';
+import Modal from 'jsx/Modal';
+
+/**
+ * React component used to display
+ * issue_tracker attachments list.
+ */
+class AttachmentsList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      attachments: this.props.attachments,
+      showModalAttachmentDelete: false,
+      deleteItem: {
+        ID: '',
+        file_name: '',
+        file_hash: '',
+      },
+    };
+    this.deleteAttachment = this.deleteAttachment.bind(this);
+    this.openModalAttachmentDelete = this.openModalAttachmentDelete.bind(this);
+    this.closeModalAttachmentDelete = this.closeModalAttachmentDelete.bind(this);
+    this.displayAttachmentOptions = this.displayAttachmentOptions.bind(this);
+  }
+
+  /**
+   * Sends DELETE request to "soft" delete
+   * the attachment selected from the user.
+   */
+  deleteAttachment() {
+    const state = Object.assign({}, this.state);
+    const url = loris.BaseURL +
```

Comment:
BaseURL should be passed as a prop, not from the global (same with all the other instances)

Reply:
@driusan fixed in latest commits